source/v5/mqtt5_options_storage.c

```diff
@@ -1743,30 +1743,6 @@ static int s_aws_mqtt5_packet_publish_view_validate_vs_connection_settings(
             return aws_raise_error(AWS_ERROR_MQTT5_PUBLISH_OPTIONS_VALIDATION);
         }
 
-        if (publish_view->topic_alias != NULL) {
-            const struct aws_mqtt5_client_options_storage *client_options = client->config;
-            if (client_options->topic_aliasing_options.outbound_topic_alias_behavior != AWS_MQTT5_COTABT_USER) {
```

Comment:
Trivial: Not erroring here seems fine but should we log that we're ignoring?

Reply:
I feel like encoder logging gives a definitive answer to what we do alias-wise.